db-utils/src/main/kotlin/com/icerockdev/common/database/sql/ColumnType.kt

```diff
@@ -20,7 +18,7 @@ class GeographyPointColumnType(private val length: Int = 4326) : ColumnType() {
     override fun setParameter(stmt: PreparedStatement, index: Int, value: Any?) {
         val obj = PGobject()
         obj.type = "geography"
-        obj.value = value.toString()
+        obj.value = value as String?
```

Comment:
Why this changed?

Reply:
For supporting null values

Reply:
but here `Any` - it can be not `String`...
`value?.toString()` more correct i think